tls/s2n_record_write.c

```diff
@@ -170,9 +189,6 @@ int s2n_record_write_protocol_version(struct s2n_connection *conn, struct s2n_st
         record_protocol_version = MIN(record_protocol_version, S2N_TLS10);
     }
 
-    /* In accordance to TLS 1.3 spec, https://tools.ietf.org/html/rfc8446#section-5.1
-     * tls record version should never be greater than 33 (legacy TLS 1.2 version).
-     */
     record_protocol_version = MIN(record_protocol_version, S2N_TLS12);
```

Comment:
nit: It might make sense to remove the line break before this line, to demonstrate that the above RFC comments also apply to this.